tests/e2e/pageobjects/ide/plugins/OpenshiftPlugin.ts

```diff
@@ -21,20 +21,20 @@ export enum OpenshiftAppExplorerToolbar {
     ReportExtensionIssueOnGitHub = 'Report Extension Issue on GitHub',
     RefreshView = 'Refresh View',
     SwitchContexts = 'Switch Contexts',
-    LogIntoCluster = 'Login into Cluster'
+    LogIntoCluster = 'Login into Cluster',
 }
 
 export enum OpenshiftContextMenuItems {
     NewComponent = 'New Component',
-    Push = 'Push'
+    Push = 'Push',
```

Comment:
@makambalaji this doesn't look right. There are no more variables after this one, the `,` shouldn't be there.

Reply:
fixed with the linter code change